lnrpc/walletrpc/walletkit.proto

```diff
@@ -79,6 +79,28 @@ service WalletKit {
     */
     rpc ListAddresses (ListAddressesRequest) returns (ListAddressesResponse);
 
+    /*
+    SignMessageWithAddr returns a signature with the private key of the provided
+    address. The address lookup in the wallet will consider the default account
+    but can be changed with a cmd argument. In addition the address type is
```

Comment:
> In addition the address type is needed to safe type and not check the address for all possible types. In case the address type and the address do not match, no other types are considered

Is this comment still relevant? The type is not part of the request. (same for verifymsg)

Reply:
Will fix, same as above 👍